test/run-tests

```diff
@@ -539,36 +519,41 @@ def choose_task(gh, repos, images, config, ansible_id, variant):
     """
 
     for owner, repo in repos:
-        pulls = gh.get(f"repos/{owner}/{repo}/pulls").json()
-        random.shuffle(pulls)
+        pyghrepo = pygh.get_repo(f"{owner}/{repo}")
+        pulls = pyghrepo.get_pulls(state='open')
+        # random.shuffle(pulls) --- cannot shuffle PaginatedList
```

Comment:
Can we convert this into a regular python list, then do the shuffle? e.g. `pulls = list(pyghrepo.get_pulls(state='open'))` ?

Reply:
Your proposed code works!
I'm wondering if there are more than one page size (30 by default), list function still works or not... What do you think? Or it's hard to imagine there are more than 30 open pull requests in a repo?

Reply:
> Your proposed code works!
> I'm wondering if there are more than one page size (30 by default), list function still works or not...

I don't know - you'll have to look at the source code to see if it is implemented as a generator or something like that.

> What do you think? Or it's hard to imagine there are more than 30 open pull requests in a repo?

Easier to imagine that github decides to change the page size to 10 or something like that :P

That is - we should make sure it works, regardless of the page size.

In the run-tests script, there are other objects we retrieve from github that may require paging - like statuses and comments - do all of these use the same page size?  It should be pretty easy to create more than 30 comments in a PR and see how paging works with comments.

Reply:
Sorry, my question was not accurate. It was just about `list(pyghrepo.get_pulls(state='open'))`. Other PaginatedList's are handled in the loop, so there should not be an issue regardless of the page size or page counts.
But probably, it might be a good idea to add a converter function from PaginatedList to List...

Reply:
Worst case, you have to do an explicit loop e.g.
`pulls = [pr for pr in pyghrepo.get_pulls(state='open')]`
that will presumably page through all of the pages and return you a `list` which contains all of the prs from all of the pages.